lib/tasks/stack.rake

```diff
@@ -159,6 +159,7 @@ namespace :stack do
       end
 
       Cluster::Stack.start_all(num_workers)
+      puts "Tagging ec2 instances and volumes"
       Cluster::Instances.create_custom_tags
```

Comment:
Hi @lbjay, the lib/cluster/instances.rb "create_custom_tags" function is being removed. Where is this one from?

Reply:
thank you. yeah i'm realizing i missed a few other things. my rubymine ide was not doing the correct code search.